de.xwic.appkit.webbase/src/de/xwic/appkit/webbase/editors/mappers/YesNoRadioGroupMapper.java

```diff
@@ -53,6 +53,9 @@ public void loadContent(IEntity entity, RadioGroup text, Property[] property) th
         if(value instanceof Boolean) {
             final Boolean booleanValue = (Boolean) value;
             text.setSelectedKey(booleanValue ? EYesNoRadioBuilder.KEY_YES : EYesNoRadioBuilder.KEY_NO);
+        } else if(value instanceof String) {
+            final String strValue = (String) value;
+            text.setSelectedKey(strValue);
```

Comment:
I am not sure how "safe" this is. You may end up with setting a key that doesnt exist. How about something like this:

```java
text.setSelectedKey("1".equals(value) || "true".equals("value") ? EYesNoRadioBuilder.KEY_YES : ...KEY_NO);
```